### PR TITLE
Rewrite/update readme

### PR DESCRIPTION
**Pull request by Liamsi**

### CODE REVIEW

File: Deterlab.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Afterwards you only need to unlock your SSH-agent / keychain once (per 
+session) and can access all your stored keys without typing the 
+passphrase each time.
+
+**OSX:**
+
+You can store your SSH-key directly in the OSX-keychain by executing:
+
+```
+$ /usr/bin/ssh-add -K ~/.ssh/<your private ssh key>
```
Comment (ineiti) [inline Deterlab.md]: I thought that \`\`\` already implies this is a command, so is it OK to add the '$' here?

Comment (Liamsi) [inline Deterlab.md]: Yes, ```  implies a code block. It doesn't really say which code though.  '$' is often used to describe [a shell prompt](https://en.wikibooks.org/wiki/Linux_For_Newbies/Command_Line#Shell_Prompt). I can remove it as it is quite clear from  the context. There isn't really right or wrong here.

Comment (ineiti) [inline Deterlab.md]: Just to avoid the "I tried to run `$ /usr/bin` and it didn't work" comment...

**Comment by ineiti** [comment]
Does that include #261?

**Comment by Liamsi** [comment]
Yes, i've updated the (missing file) and it should work out of the box now (with golang properly installed). Bit I didn't test it yet.

File: Deterlab.md (hunk 1 above)
Comment (Liamsi) [inline Deterlab.md]: done

File: Deterlab.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+simpler when deploying to DeterLab:
+
+* `-nobuild`: don't build any of the helpers which is useful if you're 
+working on the main code
+* `-build "helper1,helper2"`: only build the helpers, separated by a 
+",", which speeds up recompiling
+* `-range start:end`: runs only the simulation-lines including `start` 
+and `end`. 
+Counts from 0, start and end can be omitted and represent beginning and 
+end of lines, respectively.
```
Comment (ineiti) [inline Deterlab.md]: *beginning and end of lines* -> *first and last line of simulation*

File: Deterlab.md (hunk 1 above)
Comment (ineiti) [inline Deterlab.md]: Heh - you changed it in the main README.md, but not in this README.md...

**Comment by ineiti** [comment]
+1 for merge

**Pull request merged**